testsuite/CHANGELOG

```diff
@@ -17,6 +17,7 @@ and https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests
          tyler.je.reddy
 
   * 0.16
+    - Added unit tests for MDAnalysis.analysis.psa exception handling
```

Comment:
I'm a huge fan of these little PRs to get coverage up, just we don't need to record every single one.  I think the CHANGELOG is better used to detail changes to the testsuite internals, things like plugins and fixtures.